src/Stratis.SmartContracts.IntegrationTests/NodeExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+using Stratis.Bitcoin.Features.PoA.Voting;
+using Stratis.Bitcoin.Features.SmartContracts.PoA;
+using Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers;
+using Stratis.SmartContracts.Tests.Common.MockChain;
+
+namespace Stratis.SmartContracts.IntegrationTests
+{
+    public static class NodeExtensions
```

Comment:
Would probs prefer this namespace / type was a little more specific to suggest it's really only for whitelist networks